### ISSUE TITLE
Intermittent test failure on Windows - lib_tests.FilePath_Exists

### ISSUE THREAD

**Issue by jthelin** [jthelin/blist#35]
The `lib_tests.FilePath_Exists` test case fails intermittently when run on Windows.

```
<<<< Exit_ :: - unknown file: error: SEH exception with code 0xc0000005 thrown in the test body.
```
https://swtech.visualstudio.com/GitHub-Builds/_build/results?buildId=3213&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=f33089cd-d3fe-578c-2c5e-4ee560e10a0d&l=130

More into on SEH (Structured Exception Handling) exception:

https://stackoverflow.com/questions/13157671/seh-exception-with-code-0xc0000005-thrown-in-the-test-body

**Comment by jthelin** [comment]
This is suspicious, from latest PR build - not the "garbage"characters instead of the expected method name.

``` text
1: [ RUN      ] file_filter_tests.ReadChars
1: >>>> Enter file_filter_tests::ReadChars
1: 2371 characters read.
1: <<<< Exit_ :: - @\
1: 0Z
1: �Y
1: [       OK ] file_filter_tests.ReadChars (0 ms)
```

**Comment by jthelin** [comment]
Problem should be fixed by PR #36
